lib/data/user.ts

```
@@ -62,21 +74,24 @@ export const getUsersWithMailboxAccess = async (mailboxId: number): Promise<User
+    const role = user.mailboxRole ?? user.rawMetadata?.mailboxAccess?.[mailboxId]?.role ?? "afk";
+    const keywords = user.mailboxKeywords ?? user.rawMetadata?.mailboxAccess?.[mailboxId]?.keywords ?? [];
     const permissions = user.permissions ?? "member";
 
     return {
       id: user.id,
       displayName: user.displayName || user.rawMetadata?.display_name || "",
       email: user.email ?? undefined,
-      role: access.role,
-      keywords: access?.keywords ?? [],
+      role: role as UserRole,
+      keywords: keywords,
```

ESLint

[error] Expected property shorthand.

(object-shorthand)